### ISSUE TITLE
Native playback

### ISSUE THREAD

**Issue by Fryingpan77** [whmcs-smarters/IPTV-Smarters-TV-Demo-APK#162]
Removing native for playback has caused problems. Software decoding is always stuttering, even on powerful devices like Nvidia shield, so less powerful devices will never work. Hardware decoder is better but not great for less powerful devices either. The best implementation for this was about 2 or 3 versions back when native was default and aspect ratio changes to movies didn't effect live TV performance.

**Comment by lookitsaaron** [comment]
Yes 2.0 Built in player has a lot of problems. Last version 1.6 was better. Please fix.
